### PR TITLE
(4407) changed default value of Sliding navbar for mobile extension |@todo header

### PR DESCRIPTION
**Pull request by 4reeman**
https://redmine.devbranch.work/issues/4407

### CODE REVIEW

File: web/themes/custom/fifth_sub_bootstrap/fifth_sub_bootstrap.theme [hunk 1]
```diff
@@ -12,4 +12,8 @@ function fifth_sub_bootstrap_preprocess_node(&$variables) {
 //  $query = \Drupal::entityQuery();
 //  $query = \Drupal::database()->select()
   $variables['comment_count'] = 100500;
+
+}
+function fifth_sub_bootstrap_preprocess_page(&$variables) {
+  $variables['navbar_collapse_class'] = "offcanvas offcanvas-start";
```
Comment (devbranch-vitaliy) [inline web/themes/custom/fifth_sub_bootstrap/fifth_sub_bootstrap.theme]: Install CodeSniffer and check Drupal coding standards in the file, please.

File: web/themes/custom/fifth_sub_bootstrap/fifth_sub_bootstrap.theme [hunk 2]
```diff
@@ -12,4 +12,8 @@ function fifth_sub_bootstrap_preprocess_node(&$variables) {
 //  $query = \Drupal::entityQuery();
 //  $query = \Drupal::database()->select()
```
Comment (devbranch-vitaliy) [inline web/themes/custom/fifth_sub_bootstrap/fifth_sub_bootstrap.theme]: Dead code. Remove it.
```suggestion
```

File: web/themes/custom/fifth_sub_bootstrap/src/scss/custom/components/main_header/_main_header.scss [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  transition: all 0.5s ease-in-out;
+  transform: translate(0,0);
+}
+//{
+//  transform: translate(-20px, 10px);
+//}
+//{
+//  transition: all 0.5s ease-in-out 0s;
+//  box-shadow: rgba(0, 0, 255, 20%) 12px 12px 2px 1px;
+//}
```
Comment (devbranch-vitaliy) [inline web/themes/custom/fifth_sub_bootstrap/src/scss/custom/components/main_header/_main_header.scss]: Dead code. Remove it.
```suggestion
```

**Review by devbranch-vitaliy — CHANGES_REQUESTED** [review]

**Review by 4reeman — COMMENTED** [review]
file was changed

**Review by devbranch-vitaliy — APPROVED** [review]

**Pull request merged**